lib/modules/manager/gomod/artifacts.ts

```diff
@@ -111,6 +114,107 @@ function useModcacherw(goVersion: string | undefined): boolean {
   );
 }
 
+interface ExtraDep {
+  depName: string;
+  currentValue: string;
+  newValue: string;
+}
+
+function getExtraDeps(
```

Comment:
can we move these helper function to a separate file? this file is already very big.